src/popup/components/dashboard/Card.vue

```diff
@@ -20,12 +21,25 @@
       </div>
       <slot />
     </div>
+    <div
+      v-if="closable"
+      class="card-close"
+    >
+      <BtnPlain @click="$store.commit('hideCard', closable)">
+        <Close />
+      </BtnPlain>
+    </div>
   </div>
 </template>
 
 <script>
+import { mapState } from 'vuex';
+import BtnPlain from '../buttons/BtnPlain.vue';
+import Close from '../../../icons/times-circle.svg?vue-component';
+
 export default {
   name: 'Card',
+  components: { BtnPlain, Close },
```

Comment:
Please put each component in separate lines to avoid conflicts in the future changes.